### PR TITLE
Event Status.

### PR DESCRIPTION
**Pull request by pandilladivya**
Work Done:

-> Add "Require Volunteers" label only to the events that require volunteers.
-> Add "Registered" label to the events the user has registered to.
-> Don't allow the user to Register to the same event twice.
-> Change the event status in mapview by changing color of the marker.

The following is the video demo.
ezgif com-video-to-gif 14

@sandarumk @HimanshiJain Kindly review

### CODE REVIEW

**Comment by pandilladivya** [comment]
1. Okay. Will do.
2. That is a good idea. But don't you think the user would get confused if he see more markers?
3. Okay. We can. I'll change.
4. That commit is not included in this PR.

**Comment by HimanshiJain** [comment]
2. Possible, so how about adding a key to the map? And confusion would be less if the user can permanently see the event name on each marker.

**Comment by pandilladivya** [comment]
But since the events are mocked right now, they are not close. but if the events are near by it will be really messy if all the info on the marker id permanently seen. Also the requirement docs says that the summary of the event is shown only on click.
Ref: https://github.com/systers/volunteers-android/issues/20

**Comment by HimanshiJain** [comment]
Right, it will be cluttered. But the user should be able to locate one particular event right? So adding Google maps location link to event details page will be comfortable I suppose.

**Comment by pandilladivya** [comment]
Yes. I agree.

**Comment by pandilladivya** [comment]
@HimanshiJain Disabling the button is giving me more confusion that If I click that button, Nothing happens. Instead should I remove the button? @sandarumk Please state your views on this.
Thanks :)

**Comment by pandilladivya** [comment]
@HimanshiJain @sandarumk 
Added attendee type.
Ex: " Registered as Volunteer" "Registered as student" etc.

The following is the screenshot
screen shot 2017-08-20 at 4 18 04 pm

**Comment by HimanshiJain** [comment]
@pandilladivya So, when disabled, we can change its colour to a faded colour than original right? And set it non clickable so that the user knows it is not clickable.

**Comment by pandilladivya** [comment]
Okay.

**Comment by pandilladivya** [comment]
@HimanshiJain @sandarumk Done with the changes.
Kindly review

**Comment by pandilladivya** [comment]
@sandarumk @HimanshiJain 
Resolved conflicts.
Kindly review 😊

File: app/src/main/java/vola/systers/com/android/adapter/EventListAdapter.java [hunk 1]
```diff
@@ -56,6 +59,19 @@ public View getView(int position, View convertView, ViewGroup parent) {
         viewHolder.date.setText(event.getStartDate()+" to "+event.getEndDate());
         viewHolder.time.setText(event.getStartTime()+" to "+event.getEndTime());
         viewHolder.location.setText(event.getLocationName());
+        viewHolder.status.setText(event.getStatus());
+        if(event.getStatus().equals("Require Volunteers"))
+        {
+            viewHolder.status.setTextColor(Color.parseColor("#F89728"));
+        }
+        else if(event.getStatus()!="")
```
Comment (HimanshiJain) [inline app/src/main/java/vola/systers/com/android/adapter/EventListAdapter.java]: Add comments here, as to what cases are covered in this if case.

Comment (pandilladivya) [inline app/src/main/java/vola/systers/com/android/adapter/EventListAdapter.java]: Okay

**Pull request merged**